pytorch_to_returnn/import_wrapper/wrap.py

```diff
@@ -14,6 +14,11 @@ def wrap(obj, *, name: str, ctx: WrapCtx):
     return obj
   if isinstance(obj, ctx.keep_as_is_types):
     return obj
+  if isinstance(obj, Hashable) and obj in ctx.explicit_wrapped_objects:
+    func, args = ctx.explicit_wrapped_objects[obj]
+    args = [make_wrapped_function(arg, name=name, ctx=ctx) if isinstance(arg, _FuncTypes) else arg for arg in args]
+    wrapped_class = make_wrapped_class(func, name=name, ctx=ctx)
+    obj = wrapped_class(*args)
```

Comment:
This is too specific. `explicit_wrapped_objects` would simply give you a function which does that.
```suggestion
  if isinstance(obj, Hashable) and obj in ctx.explicit_wrapped_objects:
    func = ctx.explicit_wrapped_objects[obj]
    obj = func(obj, name=name, ctx=ctx)
```